plugins/topology/src/models.ts

```diff
@@ -190,4 +214,6 @@ export const resourceModels = {
   [TaskRunModel.kind]: TaskRunModel,
   [CheClusterModel.kind]: CheClusterModel,
   [VirtualMachineModel.kind]: VirtualMachineModel,
+  [VirtualMachineInstanceModel.kind]: VirtualMachineInstanceModel,
+  [TemplateModel.kind]: TemplateModel,
```

Comment:
If you are planning to use these resources, you'll need to update the Readme to include them as k8s custom resources.

Reply:
We are not fetching template , but just we are showing the name , abbreviation and color .